site/content/docs/5.0/components/skeletons.md

```diff
@@ -35,7 +35,7 @@ In this example we have built a fancy HTML card using Bootstrap components. Then
     {{< placeholder width="100%" height="180" class="card-img-top" text="false" >}}
     <div class="card-body">
       <h5 class="card-title">
-        <span class="skeleton-line skeleton-glow w-50"><span class="skeleton-dash bg-secondary"></span></span>&nbsp;
+        <span class="skeleton-line skeleton-glow w-50"><span class="skeleton-dash bg-secondary"></span>&#8232;</span>
```

Comment:
Also, I'm not sure this is the proper solution here @patrickhlauke

Reply:
It was a hotfix to pass the checks! I didn't know exactly what to do...